js/excellent.js

```diff
@@ -46,8 +46,11 @@
      * which may be a function name or a context reference.
      */
     excellent.Parser.prototype.autoCompleteContext = function(partialExpression) {
-        if (this.isInStringLiteral(partialExpression)) {
-            return null;
+
+        if (this.expressionContext(partialExpression) && this.expressionContext(partialExpression).match(/[(]*[^"]*["]/)) {
```

Comment:
Should save the result of calling `this.expressionContext(partialExpression)` rather than calling it twice as it is expensive.

But am also a bit confused about how this is being used. The original idea was that one would call `autoCompleteContext` on the result of `expressionContext`, e.g.

```js
var currentExpression = parser.expressionContext(beforeCaret);
if (currentExpression) {
  var currentIdentifier = parser.autoCompleteContext(currentExpression);
}
```

Which means the string literal check in `autoCompleteContext` is only checking within an existing expression and can't include `"@SUM(...)` for example. Seems we don't use it this way?